omp/__init__.py

```diff
@@ -81,38 +61,43 @@ def init_not_msvc(self):
 
         paths = []
         for env_var in env_vars:
-            paths += os.environ.get(env_var, '').split(os.pathsep)
+            env_paths = os.environ.get(env_var, '')
+            if env_paths:
+                paths.extend(env_paths.split(os.pathsep))
+
 
         libomp_names = self.get_libomp_names()
+
         for libomp_name in libomp_names:
-            if cxx is None or sys.platform == 'win32':
-                # Note: Clang supports -print-file-name, but not yet for
-                # clang-cl as of v12.0.0 (April '21)
-                continue
 
-            cmd = [cxx, '-print-file-name=' + libomp_name]
-            # the subprocess can fail in various ways in that case just give up
+            cmd = [cxx, '-print-file-name=' + libomp_name +
+                   self.get_shared_lib_extension()]
```

Comment:
so this invocation does not contain the `lib` prefix but _does_  contain the extension, which seems like the neither-nor case in terms of what's a useful variant...?